### ISSUE TITLE
How to specify to use daily streams vs release streams?

### ISSUE THREAD

**Issue by xnox** [canonical/multipass#1416]
When building a core18 project with snapcraft, how do I specify to launchp multipass VM from "daily" streams instead of "release" stream?

(ie. /daily/ vs /releases/ in the urls, with like focal production names, which at the moment are only published in /daily/ but not yet in /releases/)

**Comment by townsend2010** [comment]
I believe this is more of a Snapcraft question since they are the ones who call Multipass in this case.  @sergiusens, any ideas?

**Comment by townsend2010** [comment]
Wait, after reading #1417, I'm a little lost on what you are asking here.  Are you referring to the cloud-images, minimal images, or the new buildd images we've been working towards?

**Comment by Saviq** [comment]
Yeah I was meaning to reply here. @xnox we harcode what `snapcraft:core*` means in terms of the image being launched. With the images popping up in https://cloud-images.ubuntu.com/buildd/ we might improve that, provided they are tagged appropriately.

That said, in #756 we designed how you could affect that.

**Comment by xnox** [comment]
In LXD, I can do:
    lxd launch ubuntu:bionic
    lxd launch ubuntu-daily:bionic

First one will launch latest bionic from the release stream from https://cloud-images.ubuntu.com/releases/streams/

The second one will launch latest bionic from the daily stream from https://cloud-images.ubuntu.com/daily/streams/

I've now discovered that 18.04 is a shortcut to "release:18.04", and that `multipass find release:` and `multipass find daily:` seems to be equivalent names to the lxd remote images.

I found that out by guessing.

Is there something equivalent to `lxc remote list` to see all the valid multipass prefixes? I.e. something that will output `snapcraft release daily` prefixes?
